src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -59,7 +59,7 @@
     <Compile Include="System\Runtime\CompilerServices\IntrinsicAttribute.cs" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' != '$(NetFrameworkMinimum)' and '$(IsPlaceholderTargetFramework)' == 'true'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
This is fine, it would have also been OK to change to `'$(TargetFramework)' != '$(NetFrameworkMinimum)' and '$(IsPlaceholderTargetFramework)' != 'true'`

Reply:
Good catch. I mistakenly both were != originally. Since both conditions mean the same, I think I prefer to leave it as it is now since it's more explicit.